lib/openhab/dsl/monkey_patch/ruby/time.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module OpenHAB
+  module DSL
+    module MonkeyPatch
+      module Ruby
+        #
+        # Extend Time to make to_s return string parseable by OpenHAB
+        #
+        module TimeExtensions
+          include OpenHAB::Core
+
+          #
+          # Convert to ISO 8601 format
+          #
+          # @return [Java::JavaTime::Duration] Duration with number of units from self
+          #
+          def inspect
```

Comment:
I think this should be to_s and not inspect.

Reply:
Inspect should be deeper than just the openhab time representation.

Reply:
Had the same thought, will change to only override to_s an leave inspect to the default.